daprdocs/content/en/operations/hosting/kubernetes/cluster/setup-gke.md

```diff
@@ -0,0 +1,55 @@
+---
+type: docs
+title: "Setup a Google Kubernetes Engine cluster"
+linkTitle: "Google Kubernetes Engine"
+weight: 2000
```

Comment:
Looks like another page in this folder has the value "2000". This is used to order the articles in the directory so these should have different values. Please change this value to "3000"

Reply:
Changed to 3000 :+1:

I see both `setup-aks.md` and `setup-minikube.md` in this folder have a weight of 2000. Do you mind me changing the weight for the minikube page to 1000 in this PR or would you prefer a separate PR?

Reply:
Thanks for catching that and fixing :)